### DISCUSSION TITLE
exporing-the-code-behind-ihttpclientfactory/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#51] [Announcements]
# exporing-the-code-behind-ihttpclientfactory/

In this post I look in depth at the code behind the default implementation of IHttpClientFactory and see how it manages HttpClient handler lifetimes.

https://andrewlock.net/exporing-the-code-behind-ihttpclientfactory/

**Comment by wangengzheng** [comment]
### Great article thanks for the analysis

[a comment by an automated account is not shown]

**Comment by V-asakot** [comment]
Thank you, great article
